newIDE/app/src/ObjectsList/index.js

```diff
@@ -1363,28 +1393,16 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
 
     return (
       <Background maxWidth>
-        <Column>
```

Comment:
Removing the column, you removed the margin, you should maybe remove the `noMargin` in the child Column